cross/prowlarr/Makefile

```diff
@@ -1,13 +1,15 @@
 PKG_NAME = Prowlarr
-PKG_VERS = 0.4.10.2111
+PKG_VERS = 1.1.3.2521
 PKG_EXT = tar.gz
-PKG_DIST_NAME = $(PKG_NAME).develop.$(PKG_VERS).linux-core-$(PKG_DIST_ARCH).$(PKG_EXT)
+PKG_DIST_NAME = $(PKG_NAME).master.$(PKG_VERS).linux-core-$(PKG_DIST_ARCH).$(PKG_EXT)
 PKG_DIST_SITE = https://github.com/Prowlarr/Prowlarr/releases/download/v$(PKG_VERS)
 PKG_DIR = $(PKG_NAME)
 
 DEPENDS = cross/curl cross/sqlite
 
-UNSUPPORTED_ARCHS = $(DOTNET_UNSUPPORTED_ARCHS)
+UNSUPPORTED_ARCHS = $(PPC_ARCHS) $(ARMv5_ARCHS) $(ARMv7L_ARCHS) armada370
+# custom exclusions for dotNET 6 on ARMv7 archs
+UNSUPPORTED_ARCHS_TCVERSION = armv7-6.1
 
```

Comment:
if this is all the same for the *ARR packages, we probably can define something like
`DOTNET_ARR_ARCHS = 1` here.

and in `spksrc.archs.mk`
```
ifeq ($(strip $(DOTNET_ARR_ARCHS)),1)
    UNSUPPORTED_ARCHS = $(PPC_ARCHS) $(ARMv5_ARCHS) $(ARMv7L_ARCHS) armada370
    # custom exclusions for dotnet on ARMv7 archs
    UNSUPPORTED_ARCHS_TCVERSION = armv7-6.1
endif
```


BTW: _I prefer the name `dotnet` over .NET, dotNET, dotnet core, as it is the name of the cli._

Reply:
@hgy59 thanks for the suggestion. I've tested it and it works! I've also included specific variables for dotnet core and servarr since the main difference is the support for x86. Also, I note that we previously had the `UNSUPPORTED_ARCHS` variable in both the 'cross' makefile and the 'spk' makefile. I believe we should standardise this. I have opted to put this into the 'spk' makefile similar to what we have for jackett.

Now all that being said, I foresee a problem merging this. I was waiting on the merging of #5592 which fixes the installer call function but now that will also depend on this. So the only way forward I see is to extract the framework specific fixes into a new PR and remove them from the package PRs. I'll get started on that so that the framework specific PR can be merged before we do the package ones.

How does this all sound to you?

Reply:
> I note that we previously had the `UNSUPPORTED_ARCHS` variable in both the 'cross' makefile and the 'spk' makefile. I believe we should standardise this. I have opted to put this into the 'spk' makefile similar to what we have for jackett.

For dotnet this might be ok, particularly when downloading prebuilt packages and with single dependency spk/xy -> cross/xy.
But for regular cross packages, we trust on the information in the cross Makefile, to know which package limits the supported archs (those might be deep nested like cross/ffmpeg -> cross/libass -> cross/harfbuzz -> cross/cairo -> .... and harfbuzz does not support gcc >= 4.9).

Reply:
Okay, I'll keep that in mind. I'll use the new variable in the 'spk' makefile but exclude from the 'cross' makefile only for the `*arr` packages which are based on pre-built downloads with single dependencies.

Speaking of, I note that some `*arr` packages have some dependencies listed in the 'cross' and some in the 'spk' while others have none in the 'cross' and all in the 'spk'. Any preference for these? Also is there an easy way to know which are actually needed? For example some include 'curl' while others do not.